_articles/importing/import-from-1password.md

```diff
@@ -7,47 +7,61 @@ popular: false
 tags: [import, 1password]
 ---
 
-Importing your data from 1Password into Bitwarden is easy. 1Password has two versions of their application: 1Password 4 and 1Password 6 and 7. Depending on which operating system and version of 1Password and you are using, follow the proper steps below.
+To migrate from 1Password to Bitwarden, export your data from 1Password and use the Bitwarden Import Tool to quickly fill your Vault with logins.
 
-## Export Your 1Password 1pif Logins
+Export your data from 1Password as a `.1pif` or `.csv`, depending on which version and operating system you are using:
 
-{% callout info %}
-This set of instructions is only for:
+- `.1pif` files can be exported by **1Password 4 on Windows**, **1Password 6 on macOS**, and **1Password 7 on macOS** (*latest*).
```

Comment:
windows is capable of exporting 1pif.

I'm also not clear on what the (*latest*) tag is buying us here. I don't know how far back the 1pif format has been supported. Is it trying to say that 1password 7 is the latest version?

Reply:
Windows running v7? I based this list on it not being an option for 1PW Windows here: https://support.1password.com/export/. Admittedly, I don't have a Windows machine to test it on.

Reply:
yup, don't know why 1pass's documentation is wrong, but here's my windows desktop version 7.6.785 offering to save 1pif

![image](https://user-images.githubusercontent.com/18214891/101540679-f764f180-3965-11eb-80ad-cabf781c63b5.png)